source/api/extension/text-granularity/index.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Text Granularity Extension
+layout: spec
+tags: [extension, granularity]
+cssversion: 2
+---
+
+## Status of this Document
+{:.no_toc}
+
+This document is not subject to [semantic versioning][notes-versioning].
+Changes will be tracked within the document.
+
+{% include copyright.md %}
+
+## Table of Contents
+{:.no_toc}
+
+* Table of Discontent (will be replaced by macro)
+{:toc}
+
+## 1. Introduction
+
+### 1.1 Objectives and Scope
+
+One use of annotation in IIIF is to associate text with IIIF resources, with the text originating from optical character recognition (OCR) software, manual transcription, existing digitized text, or other sources. Furthermore, it is a common practice to produce sets of annotations at a particular level of ‘text granularity’; that is to say that each annotation in the set references the same unit of text, such as a character, line, or page. This extension recommends a pattern for indicating the level of text granularity of an annotation.
+
+### 1.2 Motivating Use Cases
+
+A number of common workflows can result in annotation sets with discrete levels of text granularity:
+
+- OCR text is frequently available at a range of granularities, from page to character.
+- Manual transcription user interfaces may constrain user input to the level of the line,
+block, or page.
+- Transcriptions may be produced without coordinate data and thus have a very coarse level of granularity, such as page- or block-level.
+
+Identification of the level of text granularity in published annotations can facilitate the reuse of their textual content and target regions in other applications. Search is a primary use case, with the word-level annotations allowing for the accurate highlighting of search result terms in the user interface. Software designed for user interaction and input, such as an application designed to allow users to correct OCR text, might suppress finer-grained levels and make use of paragraph- or block-level text to simplify editing.
```

Comment:
>Search is a primary use case, with the word-level annotations allowing for the accurate highlighting of search result terms in the user interface.

This bothers me a little because it leads with a suggestion that could be read as the spec being concerned with server-side tech.  Could we reverse and say something are:

> Interfaces that require highlighting of search results in context are a primary use case.

?

Reply:
>Software designed for user interaction and input, such as an application designed to allow users to correct OCR text, might suppress finer-grained levels and make use of paragraph- or block-level text to simplify editing.

Seems like a little bit of a non-sequitur? Previous sentence is a use case; this seems more like an implementation detail?

Reply:
Reworked to emphasize these are use cases. 

>  A primary use case is search: clients could use word-level transcription annotations to provide a search function capable of accurate hit highlighting.  Other common use cases derive from crowdsourced text input, such as the correction of OCR transcription, which may require the user to work with a text at a single level of granularity, such as line- or block-level.